jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -318,12 +352,33 @@ def vue_update_subset(self, *args):
             if len(self.subset_states) <= index:
                 return
             sub_states = self.subset_states[index]
+
+            # we need to push updates to subset in pixels. to do this when wcs
+            # linked, convert the new input subset parameters from sky to pix
+            wcs = None
+
+            if self.display_sky_coordinates:
+                wcs = self.app._get_wcs_from_subset(sub_states)
+
+                if wcs is not None:
+                    # convert newly entered sky coords to pixel
+                    skyregion = utils._subset_def_to_region(self.subset_types[index], sub)
+                    new = utils._get_pixregion_params_in_dict(skyregion.to_pixel(wcs))
+                    # convert previous entered sky coords to pixel
+                    new_orig_skyregion = utils._subset_def_to_region(self.subset_types[index], sub, val='orig')
+                    new_orig = utils._get_pixregion_params_in_dict(new_orig_skyregion.to_pixel(wcs))
```

Comment:
Let's make these variable names a bit easier to parse so that future-us can figure out what they're doing. I'll make these suggestions, but you can adjust as necessary:
* `skyregion` -> `updated_sky_region`
* `new` -> `updated_pixel_region_attributes`
* `new_orig_skyregion` -> `original_pixel_region`
* `new_orig` -> `original_pixel_region_attributes`

```suggestion
                    updated_sky_region = utils._subset_def_to_region(self.subset_types[index], sub)
                    updated_pixel_region_attributes = utils._get_pixregion_params_in_dict(updated_sky_region.to_pixel(wcs))
                    # convert previous entered sky coords to pixel
                    original_pixel_region = utils._subset_def_to_region(self.subset_types[index], sub, val='orig')
                    original_pixel_region_attributes = utils._get_pixregion_params_in_dict(original_pixel_region.to_pixel(wcs))
```

Reply:
changed these